### ISSUE TITLE
Getting undefined object when requiring express-mysql-sessio

### ISSUE THREAD

**Issue by GMNetto** [chill117/express-mysql-session#39]
When I try to execute this line:
var MySQLStore = require('express-mysql-session')(session);
I am getting that it is impossible to set the proporty options of undefined.
The error happens in line 39 of index.js.

Do anyone have any idea about how to solve it?

**Comment by chill117** [comment]
@GMNetto To help you solve the issue, I'll need to the following info:
* The full error stack trace that you are seeing from your terminal window
* Node version number - `node -v`
* NPM version number - `npm -v`

**Comment by GMNetto** [comment]
node version is 5.3.0
npm version is 3.3.12
the full stack is:
...\node_modules\express-mysql-session\lib\index.js:39
        this.options = options || {};
                     ^

TypeError: Cannot set property 'options' of undefined
    at module.exports (...\node_modules\express-mysql-session\lib\index.js:39:15)
    at Object.<anonymous> (...\server.js:7:50)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:430:10)
    at startup (node.js:141:18)
    at node.js:980:3

**Comment by chill117** [comment]
Looks like you are missing the `new` keyword when creating the session store instance. Here's an example from the readme:
```js
var MySQLStore = require('express-mysql-session')(session);

var options = {
    host: 'localhost',
    port: 3306,
    user: 'session_test',
    password: 'password',
    database: 'session_test'
};

// The `new` keyword is required in the following line:
var sessionStore = new MySQLStore(options);

app.use(session({
    key: 'session_cookie_name',
    secret: 'session_cookie_secret',
    store: sessionStore,
    resave: true,
    saveUninitialized: true
}));
```

**Comment by GMNetto** [comment]
Actually the error is happening in the line:

var MySQLStore = require('express-mysql-session')(session);

**Comment by chill117** [comment]
Can you provide the full code snippet where you're using this module?

**Comment by TomMettam** [comment]
This was happening for me with v0.4.4 (which is the current stable release according to npm).  Updating to 1.0.0-rc.0 solved the issue.

**Comment by chill117** [comment]
There has been some confusion for those who were still using the 0.x branch but with the latest documentation.

I have now published v1.0.0 to npm. This should be considered the latest stable release now.
